Amazon_Linux_2023_Base/Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG BUILD_IMAGE="amazonlinux:2023"
+
+FROM ${BUILD_IMAGE}
+
+LABEL maintainer="CJSE"
+ARG DNF_PACKAGES="pip gzip tar unzip openssl shadow-utils xz wget gnupg dirmngr"
+
+RUN dnf update -y &&\
+    dnf install -y ${DNF_PACKAGES} --allowerasing
+
+# https://github.com/aws/aws-cli/issues/8036
+RUN pip install wheel &&\
+    pip install "Cython<3.0" "pyyaml<6" --no-build-isolation 
```

Comment:
This probably isn't needed - if you look at the issue, it should only affect the previous version of `awscli`.